### PR TITLE
CVEs landing page redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Added redesign as per [mock-up](https://www.figma.com/file/nUJEBpOPTW74RqDRuMQXxB/23.04-U.com---CVEs?node-id=949%3A22802&mode=dev)

## Future work

The releases links will be handled in a separate PR, as will the advanced search link.

## QA

- View the site locally in your web browser at: https://ubuntu-com-13382.demos.haus/security/cves
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the page matches the design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5145

### CODE REVIEW

**Comment by juanruitina** [comment]
One very minor comment: please make the ruler above "By Ubuntu release" the same width as the row below (9 columns) as per the mockup.

Other than that, great work. Love how the cards downscale now on smaller viewports :)

**Comment by petesfrench** [comment]
In the design, on medium screens, the button remains to the right of the search input:
image

File: webapp/security/helpers.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    """
+
+    more_packages = False
+    if len(cve["packages"]) > 1:
+        more_packages = True
+
+    for package in cve["packages"]:
+        # Check if all statuses for all packages are the same, excluding DNE and empty data
+        if (len({d["status"] for d in package["statuses"] if d["status"] not in {"DNE", ""}}) == 1):
+            #  todo: set status
```
Comment (petesfrench) [inline webapp/security/helpers.py:16]:
```suggestion
            #  todo: set status
```
do you need this still?

**Review by petesfrench — APPROVED** [review]
Couple minor comments but looks good to me

File: templates/security/cve/index.html [hunk 2]
```diff
@@ -10,82 +11,157 @@
...
+                  <i class="p-icon--placeholder"></i>
+                {% endif %}
+                <span>{{ cve.priority.split() | first | capitalize }} priority</span>
+              </span>
+            </p>
+          </div>
+        </div>
+        <div class="col-3 col-medium-2">
+          {% set status = cve.summarized_status %}
+            {% if status.name == "Some fixed" %}
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/security/cve/index.html:121]: They're set in the `get_summarized_statuses` function in the helpers.py file. The names are hard coded depending on which check is valid for the cve as a whole, so no white spaces or variations in capitalization, eg from line 55: 
```

                cve["summarized_status"] = {
                    "name" : "Some fixed",
                    "fixed_count" : total_fixed,
                    "total_count" : total_fixable
                }
```

File: templates/security/cve/index.html [hunk 3]
```diff
@@ -10,82 +11,157 @@
...
+          <p class="p-text--small-caps u-text--muted u-no-margin--top">{{ cve.packages | length}} affected packages</p>
+          <p class="u-text--muted">
+            {% for package in cve.packages %}
+              {% if cve.packages | length > 6 %}
+                {% if loop.index < 6  %}
+                  {{ package.name }}, 
+                {% elif loop.index == 6 %}
+                  and {{ cve.packages | length - 5 }} more
+                {% endif %}
+              {% elif cve.packages | length < 6 %}
```
[the remark this answers is not in the document]
Comment (mtruj013) [reply templates/security/cve/index.html:145]: Yikes lol

**Pull request merged**